### ISSUE TITLE
error http 400

### ISSUE THREAD

**Issue by joseph-vidal-rosset** [myuhe/org-gcal.el#102]
Hello, 
I am near to succeed to use org-gcal but I get always the same error message:
"Wrote /home/joseph/MEGA/org/agenda.org
(New file)
Wrote /home/joseph/MEGA/org/todo.org
Make /home/joseph/MEGA/git/scimax/userorg-gcal/.org-gcal-token [2 times]
Got error: (error http 401)
Got error: (error http 400)
(error http 400)"

Version of my emacs: 
joseph@joseph-MS-7996 ~ $ emacs --version
GNU Emacs 27.0.50

The very last one, I think... 
Your help is welcome.

**Comment by tasshin** [comment]
I am also getting these errors as well as (error http 403).

It seems that if I delete the whole file and try again, it updates accurately, but otherwise I am stuck with old calendar info.

**Comment by rollacaster** [comment]
darosio's comment here #82 fixed it for me

**Comment by jskulski** [comment]
I ran into this as well.

darosio's fix didn't work at first. I had to delete the `.emacs.d/org-cal/.org-gcal-token` and then eval'd `(org-gcal-request-token)` which worked.

**Comment by ygol** [comment]
following @dominic998 solution in #82, I could get it to work last week but since few days it doesn't work anymore. I tried deleting .emacs.d/org-gcal/.org-gcal-token and then evaluating (org-gcal-request-token) and also tried several times over these days to use org-gcal-refresh-token but I get each time an http error 401.
Anyone else has this problem?

**Comment by hughwilliams94** [comment]
Indeed, I confirm. I tried to follow @dominic998 solution in #82 and get no success. Please advise on any more info that I can provide? Just to add I get a 400 error rather than a 401

**Comment by davidszp** [comment]
Same problem. On org-gcal-refresg-token I get:

`Make c:/Users/[User]/AppData/Roaming/.emacs.d/org-gcal/.org-gcal-token
Got error: (error http 400)
(error http 400)`

**Comment by holzee** [comment]
`rm .emacs.d/org-gcal/.org-gcal-token` then eval `(org-gcal-request-token)` works for me

**Comment by larskouwenhoven** [comment]
This works for me as well - but then the problem returns the next day. Is there a more permanent solution?

**Comment by hasan-sh** [comment]
I had a similar problem and spent hours to fix the returned http error. After many trials and debugging, it was because i created *a different calendar* and its `calendar-id` was not the same as my email address; I don't know why that did/does not work?

**But** yeah so that was the problem, only when i try to create an event. For fetching, however, it worked just fine. So that's still mystery to me!
Anyway, if you use your email address as the calendar id it should work fine. 

*Another tip*: activate the alert feature; ```(setq alert-default-style 'libnotify)``` so that you get a feedback on what went good/wrong.

**Comment by telotortium** [comment]
Hi, this repo is obsolete. You should try https://github.com/kidd/org-gcal.el to see if it fixes your issues.

**Comment by telotortium** [comment]
Hi everyone! This repository is obsolete - the actively developed fork is now https://github.com/kidd/org-gcal.el, which is also the "org-gcal.el" on MELPA (see [#115](https://github.com/myuhe/org-gcal.el/issues/115)). I'm one of the maintainers of https://github.com/kidd/org-gcal.el. I can't directly close issues or PRs on this repo, but please download https://github.com/kidd/org-gcal.el and see if your issue or PR is still applicable. If so, open an issue or PR on https://github.com/kidd/org-gcal.el. In any case, you should close your issue or PR in this repo.
